include/lsst/meas/base/FluxUtilities.h

```diff
@@ -36,33 +36,35 @@ namespace meas {
 namespace base {
 
 /**
- *  @brief A reusable result struct for flux measurements.
+ *  @brief A reusable result struct for instFlux measurements.
  */
 struct FluxResult {
-    Flux flux;                 ///< Measured flux in DN.
-    FluxErrElement fluxErr;  ///< Standard deviation of flux in DN.
+    InstFlux instFlux;               ///< Measured instFlux in DN.
+    InstFluxErrElement instFluxErr;  ///< Standard deviation of instFlux in DN.
 
     /// Default constructor; initializes everything to NaN.
     FluxResult();
 
-    /// Constructor from flux and its uncertainty
-    explicit FluxResult(Flux flux_, FluxErrElement fluxErr_) : flux(flux_), fluxErr(fluxErr_) {}
+    /// Constructor from instFlux and its uncertainty
+    explicit FluxResult(InstFlux instFlux_, InstFluxErrElement instFluxErr_)
+            : instFlux(instFlux_), instFluxErr(instFluxErr_) {}
 };
 
 /**
  *  @brief A FunctorKey for FluxResult
  *
- *  This class makes it easy to copy fluxes and their uncertainties to and from records, and provides
+ *  This class makes it easy to copy instFluxes and their uncertainties to and from records, and provides
  *  a method to add the appropriate fields to a Schema.
  */
 class FluxResultKey : public afw::table::FunctorKey<FluxResult> {
 public:
     /**
-     *  Add a pair of _flux, _fluxErr fields to a Schema, and return a FluxResultKey that points to them.
+     *  Add a pair of _instFlux, _instFluxErr fields to a Schema, and return a FluxResultKey that points to
+     * them.
```

Comment:
Again, I worry about the name: if this is FluxResultKey then what do you call the corresponding object for non-inst fluxes? InstFluxResultKey seems safer, but I have no idea how disruptive such a change would be.

Reply:
If/when we eventually implement the new `_flux` fields, they could be attached to the FluxResultKey as `flux` and `fluxErr`, since they should share `flag` with `instFlux`.